notebooks/growth_model_forecast.ipynb

```diff
@@ -0,0 +1,1174 @@
+{
```

Comment:
<p>Maybe add a note here to mention and describe the Kaggle dataset we are using </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>